main.go

```diff
@@ -310,14 +312,14 @@ func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
 		Registry: registry,
 	})
 
-	if err := (&runtimecontrollers.ExtensionReconciler{
-		Client:        mgr.GetClient(),
-		RuntimeClient: runtimeClient,
-		Registry:      registry,
-	}).SetupWithManager(ctx, mgr, concurrency(1)); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "Extension")
-		os.Exit(1)
-	}
+	// if err := (&runtimecontrollers.ExtensionReconciler{
```

Comment:
If you want you can already setup this reconciler behind the feature gate. I think it should just work vs. having to deal with the linter findings

Reply:
I'll remove this code.